sympy/utilities/tests/test_lambdify.py

```diff
@@ -783,6 +783,13 @@ def test_issue_12092():
     f = implemented_function('f', lambda x: x**2)
     assert f(f(2)).evalf() == Float(16)
 
+def test_issue_13881(): # tests for #11566
+    from sympy import MatrixSymbol
+    x = MatrixSymbol('x', 3, 1)
+    f = lambdify(x, x.T*x)
+    assert f(Matrix([1, 2, 3])) == 14
```

Comment:
The test case here seems to fail.

Reply:
Yes I know but in my local terminal this test runs.

Reply:
I'm not sure why it fails here, I think you should try asking on gitter.

Reply:
Same was the issue with #13934.

Reply:
It's passing on my local terminal too.